### DISCUSSION TITLE
北京科技大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#30] [University]
> 回答来自群友

虽然不是985，但是211中算较好的几所啦，如果想来帝都但是985高攀不起可以考虑哦～～

优势学科：材料（劝退）、炼钢（劝退×2）
CS似乎还行不过斜对面就是北航emmmmmmmmm

本科生四年都是在学院路主校区啦。

**宿舍是上床下桌吗**
是的！女生四人间，男生六人间哦
（曾经在校内见到过mtf，猜测友好度大概还行啦？）
（附近租房子大概要3000元/月，单间，如果想要一个人住还是需要稍稍注意的哦～）

**教室和宿舍有没有空调**
有的！但是4月到10月才可以用啦quq
冬天有暖气，很暖和所以不需要空调取暖哦

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
没有，卫生间当层有、浴室在楼内有

**有早自习、晚自习吗**
大一有晚自习的，一般是到21:00最后一节晚课下课啦

**有晨跑吗**
也是大一有哦，不过挺水的了

**每学期跑步打卡的要求是多少公里，可以骑车吗**
没听说过

**寒暑假放多久，每年小学期有多长**
有的哦，每年上课时间固定上16+2 / 3 / 16+2 周，其余为假期，暑假寒假大概2:1吧～

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
疫情前外卖可以送到楼下

**学校附近有地铁站吗**
北走30分钟六道口，南走30分钟西土城/牡丹园，西走30分钟五道口，校东南西北门口均有班车站的
ps. 学院桥地铁站在修，鬼知道哪年能修好quq

**宿舍楼有洗衣机吗**
有

**校园网怎么样**
v6不收费，v4 1元/G

**每天断电断网吗，几点开始断**
不断网，平时23:30-6:00断电，考试周/大小假期均不断，空调不断电

**食堂价格贵吗，会吃出异物吗**
10元一顿，味道一般，吃出过异物
校园北门出门有麦当劳和肯德基，五道口、牡丹园和六道口均有购物中心，外卖也可送到。
校园内操场南北均有超市、购物、炸鸡烤肉水果等的出售摊位，如果不想吃食堂也可以在超市里买别的食物哦～～

**洗澡热水供应时间**
13:00-23:30

**校园内可以骑电瓶车吗，电池在哪能充电**
不知道，校园很小啦一般骑车就够了

**其它**
快递有快递柜，在学院桥附近取。  
图书馆有自习座位申请系统，可以提前申请的，不需要担心抢座位/没座位/占座位的情况哦～  
但是图书馆关门时间比较早啦，教学楼、逸夫楼开门时间要更长一些。

除此之外各个学院楼教室也对外开放，没课的时候可以自由自习的～这个最晚有23:00锁门的啦ww

**Comment by ryan-si** [comment]
年底学院路通地铁🤩

**Comment by yuanzhidao** [comment]
补充25届新生是8个人上下铺😭
